ratis-server/src/main/java/org/apache/ratis/server/impl/LogAppender.java

```diff
@@ -117,6 +121,16 @@ void stop() {
     public String toString() {
       return name;
     }
+
+    private boolean transLifeCycle(LifeCycle.State to) {
```

Comment:
NIT: Lets rename it to transitionLifeCycle.

Reply:
@lokeshj1703 Thanks for review. I have updated it.